HW02_A_Graph-Fails.Rmd

```diff
@@ -1,87 +1,97 @@
+
+```
+
+
+
+### Graph Fail 3
+Two mistakes in this graph. First, I wanted to make the the points slightly bolder, but changing the alpha to 2 does nothing. What does alpha do and what does setting it to 2 do? What could be done instead if I want the points slightly bigger? 
+
+Second, I wanted to move the legend on top of the graph since there aren't any points there, putting it at approximately the point/ordered pair (5, 40). How do you actually do this? Also, how do you remove the legend title ("class")? Finally, how would you remove the plot legend completely?
+```{r}
+mpg %>% 
+ggplot() + 
+  geom_point(mapping = aes(x = displ, y = hwy, color = class), size = 2) + 
+#alpha controls the transparency of the points, changing it to 2 does nothing since alpha operates on a % scale (it goes from 0 - 1)
+#changed the alpha argument to a size argument to make the points bigger
+  theme(legend.direction = "horizontal", 
+        legend.position = c(.6, .85), 
+        legend.title =   element_blank())
+#needed to change the numbers for the legend position to a 'relative position' (between 0 - 1), set the legend title to blank to remove it and put all of the legend code into one line to reduce redundancy
+#to remove the legend completely, you would set legend.position = "none"
+  
+```
+
+### Graph Fail 4
+I wanted just one smoothing line. Just one line, to show the general relationship here. But that's not happening. Instead I'm getting 3 lines, why and fix it please?
+
+```{r}
+mpg %>% 
+ggplot(mapping = aes(x = displ, y = hwy)) + 
+#needed to remove the color argument from the above line of code so the smooths aren't applied to each grouping individually
+  geom_point(aes(color = drv)) + 
+#by setting the color argument here instead, geom_smooth only acts on the data groupings as defined in the original ggplot line of code - so it fits one line because the data is one group
+  geom_smooth(se = F) #se = F makes it so it won't show the error in the line of fit
+```
+
+### Graph Fail 5
+I got tired of the points, so I went to boxplots instead. However, I wanted the boxes to be all one color, but setting the color aesthetic just changed the outline? How can I make the box one color, not just the outline? 
+
+Also, the x-axis labels were overlaping, so I rotated them. But now they overlap the bottom of the graph. How can I fix this so axis labels aren't on the graph?
+
+
+```{r}
+ggplot(data = mpg, mapping = aes(x = manufacturer, y = cty, fill = manufacturer, color = manufacturer)) + 
+#to make the entire box one color, both the fill and color arguments must be set to the same variable, I recommend against this however and would suggest deleting the color argument so that way the boxplots have a black outline and you could still see where the mean values lie on the plot
+  geom_boxplot() + 
+  theme(axis.text.x = element_text(angle = 45, vjust = .65))
+#added the vertical justification argument to control the vertical positioning of the x-axis text
```

Comment:
I like the way you addressed this problem. I didn't think about changing the vertical justification of the axis (I increased the length of the ticks, but they look terrible).